### PR TITLE
feat(text): change preset caption tag to span

### PR DESCRIPTION
**Pull request by astagnol**

### CODE REVIEW

**Review by dpellier — CHANGES_REQUESTED** [review]
Would be nice to add somewhere in the doc that the `caption` preset is only about giving a custom style, but for actual `caption` (table) or `figcaption` (image) they should use the native tag using either an ods-text inside of it or the sass mixin

File: packages/storybook/stories/components/text/text.stories.ts [hunk 1]
```diff
@@ -105,3 +106,52 @@ export const CustomCSS: StoryObj = {
...
+    <th scope="row">Chris</th>
+    <td>22</td>
+  </tr>
+  </tbody>
+</table>
+<style>
+  caption {
+    caption-side: bottom;
+  }
+  table {
```
Comment (dpellier) [inline packages/storybook/stories/components/text/text.stories.ts:134]: use a specific uniq id as this is apply to the whole page => this specifically apply on the `Properties` table on top of the page.

File: packages/storybook/stories/components/text/text.stories.ts [hunk 2]
```diff
@@ -105,3 +106,52 @@ export const CustomCSS: StoryObj = {
...
+  </thead>
+  <tbody>
+  <tr>
+    <th scope="row">Chris</th>
+    <td>22</td>
+  </tr>
+  </tbody>
+</table>
+<style>
+  caption {
```
Comment (dpellier) [inline packages/storybook/stories/components/text/text.stories.ts:131]: use a specific uniq id as this is apply to the whole page

File: packages/storybook/stories/components/text/text.stories.ts [hunk 3]
```diff
@@ -105,3 +106,52 @@ export const CustomCSS: StoryObj = {
...
+  tags: ['isHidden'],
+  render: () => html`
+<figure>
+  <img src="https://images.crunchbase.com/image/upload/c_pad,h_256,w_256,f_auto,q_auto:eco,dpr_1/ayzwkdawmlyzvuummuf4" alt="OVHcloud logo" />
+  <figcaption>
+    <ods-text preset="caption">My picture title</ods-text>
+  </figcaption>
+</figure>
+<style>
+  img {
```
Comment (dpellier) [inline packages/storybook/stories/components/text/text.stories.ts:152]: use a specific uniq id as this is apply to the whole page

File: packages/storybook/stories/components/text/text.stories.ts [hunk 4]
```diff
@@ -32,6 +32,7 @@ export const Demo: StoryObj = {
         type: { summary: 'string' }
       },
       control: { type: 'select' },
+      description: 'Caption preset is only custom styling. For `<caption>` or `<figcaption>`, use the native tag with ods-text or Sass mixin',
```
Comment (dpellier) [inline packages/storybook/stories/components/text/text.stories.ts:35]: probably not needed here

**Review by dpellier — CHANGES_REQUESTED** [review]

**Review by aesteves60 — APPROVED** [review]

**Pull request merged**